var/spack/repos/builtin/packages/pbbam/package.py

```diff
@@ -34,8 +34,8 @@ def cmake_args(self):
         return options
 
     def install(self, spec, prefix):
-        install_tree('spack-build/bin', prefix.bin)
-        install_tree('spack-build/lib', prefix.lib)
+        install_tree('../spack-build/bin', prefix.bin)
+        install_tree('../spack-build/lib', prefix.lib)
```

Comment:
For all CMake packages, Spack has defined a `build_directory` attribute, which is where it performs the build. I think it would be more straightforward to access this like:

```
install_tree(os.path.join(self.build_directory, 'lib'), prefix.lib)
```

(this will end up being the same path, but I think it will be more understandable to other users)

Reply:
Thx @scheibelp. Do we have a documentation for all the buildin attributes and predefined env variables?